### PR TITLE
Added Kevin Powell's Flexbox Layout Cheatsheet

### PR DESCRIPTION
**Pull request by CBID2**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by CBID2** [comment]
Can someone help me format this?

**Comment by eshellman** [comment]
We won't add an email-required cheatsheet

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -192,6 +192,7 @@
 * [CSS Flexbox Cheatsheet](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) - Chris Coyier (HTML)
 * [CSS Grid Cheatsheet](https://css-tricks.com/snippets/css/complete-guide-grid/) - Chris House (HTML)
 * [FLEX: A simple visual cheatsheet for flexbox](https://flexbox.malven.co) - Chris Malven (HTML)
+* [Flexbox Layout Cheatsheet](https://flexbox-pattern-cheatsheet.kevinpowell.co/) - a flexbox layout cheatsheet, Kevin Powell (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:195]: * [Flexbox Layout Cheatsheet](https://flexbox-pattern-cheatsheet.kevinpowell.co) - Kevin Powell (PDF) (* email address required*)

Comment (CBID2) [reply more/free-programming-cheatsheets.md:195]: Thanks @eshellman!

**Pull request closed without merging**